projects/fsastorefront-e2e-cypress/cypress/integration/checkout/insurance/renters-checkout.e2e-spec.ts

```diff
@@ -75,16 +75,21 @@ context('Renters Checkout', () => {
   it('Check order confirmation', () => {
     checkout.checkAccordions('rentersFinalReview');
     checkout.checkOrderConfirmation();
-    cy.wait(200000);
+    //cy.wait(200000);
```

Comment:
Are these comments left on purpose for some other test environment?

Reply:
Removed